src/QuizProps.js

```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
```

Comment:
Using axios is definitely the right way to start things off and keep them simple. If you find yourself wanting a bit more data on the state of your API call (or some fancy things like refetching) I've been playing around with [React Query](https://react-query.tanstack.com/examples/simple) and it's been awesome (and really popular in the React community). Not suggesting any re-writes but something good to keep in mind for the future 😄